src/components/Onfido/BaseOnfidoWeb.tsx

```diff
@@ -94,7 +94,7 @@ function initializeOnfido({sdkToken, onSuccess, onError, onUserExit, preferredLo
             }
             onSuccess(data);
         },
-        onError: (error: OnfidoError) => {
+        onError: (error) => {
```

Comment:
```js
import type {SdkErrorTypes} from 'onfido-sdk-ui/types/shared/Error';

type OnfidoError = {
    // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
    exception?: Error | unknown;
    type: SdkErrorTypes;
    message: string;
};
```

Reply:
I'm getting the following TS errors when deleting the `OnfidoError` type

<img width="778" alt="Screenshot 2025-02-18 at 19 35 46" src="https://github.com/user-attachments/assets/feead9ab-2326-4818-9100-275711ff1f79" />
<img width="818" alt="Screenshot 2025-02-18 at 19 35 36" src="https://github.com/user-attachments/assets/ac54459d-d4f1-415f-808a-aff0f1efccda" />

Reply:
You're right, that did the trick!
I still think there might be value in extending the type so it's consistent with actual errors thrown, but that's NAB, let's resolve the issues In https://github.com/Expensify/App/pull/56940#issuecomment-2665705600 first